site/reference/schema/kptfile/README.md

```diff
@@ -0,0 +1,140 @@
+# Example kptfile
```

Comment:
File name should the the resource kind name e.g. `Kptfile`, `FunctionResultsList`, etc.

Reply:
Done.